dist/src/main/assembly/conf/brooklyn/default.catalog.bom

```diff
@@ -30,11 +30,11 @@ brooklyn.catalog:
         docker.policy.ha.enable: false
         docker.container.strategies:
         - $brooklyn:object:
-            type: clocker.docker.location.strategy.MaxContainersPlacementStrategy
+            type: clocker.docker.location.strategy.basic.MaxContainersPlacementStrategy
```

Comment:
I think we can ignore Clocker renames of a certain age (not sure exactly what, but OSGi work is probably old enough). It's the more recent things that concern me (particularly where potential customers etc have been pointed at the download url, as well as the more forgiving community at large).